plasmapy/particles/data/test/__init__.py

```diff
@@ -3,7 +3,8 @@
 https://github.com/openPMD/openPMD-example-datasets/tree/b4f87b817629b99a048026a2724a5b265810d8be
 """
 import glob
-import os
 
-data_dir = os.path.dirname(__file__)
-data_files = glob.glob(os.path.join(data_dir, "*.[!p]*"))
+from pathlib import Path
+
+data_dir = Path(__file__).parents[0]
+data_files = glob.glob(str(data_dir.joinpath("*.[!p]*")))
```

Comment:
Don't Path objects provide their own glob method? So you wouldn't need the extension glob import nor the str conversion?

Reply:
Yes Path objects provide glob method, it returns generator so I turned it to list.